build03/default.nix

```diff
@@ -0,0 +1,7 @@
+{ self, ... }:
+{
+  flake.nixosConfiguration.build03 = self.lib.nixosSystem {
```

Comment:
`nixosConfiguration` -> `nixosConfigurations`

I would have expected this to error rather than be ignored.

Maybe we need to switch back to the ci.nix style list of attributes we were using previously to avoid this.